### DISCUSSION TITLE
In the watch demo, It looks like the field name in errors is wrong.

### DISCUSSION THREAD

**Discussion by LydiaYoon** [react-hook-form/react-hook-form#8885] [Ideas] [answered]
while studying, I found this..  
In the CodeSandbox demo on [this page](https://react-hook-form.com/api/useform/watch),  

- [JS CodeSandbox](https://codesandbox.io/s/react-hook-form-watch-v7-qbxd7)
- [TS CodeSandbox](https://codesandbox.io/s/react-hook-form-watch-v7-ts-8et1d)

It looks like the field name in `errors` is wrong.

I think `errors.age` is correct,  

```javascript
<label>Age</label>
<input type="number" {...register("age", { min: 50 })} />
{errors.age && <p>{"The number must be greater then 49"}</p>}
```
 not `errors.name`.
```javascript
<label>Age</label>
<input type="number" {...register("age", { min: 50 })} />
{errors.name && <p>{"The number must be greater then 49"}</p>}
```

cause, it is the part that shows the error related to the "age".  

### JS CodeSandbox
<img width="1100" alt="image" src="https://user-images.githubusercontent.com/44291638/185733256-821bfe2e-a9fd-4890-89df-54e4c7062672.png">  

### TS CodeSandbox
<img width="1154" alt="image" src="https://user-images.githubusercontent.com/44291638/185733314-fc9e7c5d-685f-47d8-942d-136a558f0676.png">  

what do you think about it? tell me your opinions.  
Maybe I didn't understand the intention properly.  

+) I can't speak English well, so I used a translator. 🥲

**Comment by Moshyfawn** [comment, marked as answer]
You're correct!

Thank you for spotting it; I've fixed it just now.

Good studies to you!

_P.S. don't worry worry about your English ;)_
